src/js/page_sections/LanguageSelectBar.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-static';
+import { find } from 'lodash';
+import { SUPPORTED_LANGUAGES } from 'js/i18n/constants';
+
+const selectedLanguage = (lang) => find(SUPPORTED_LANGUAGES, {'code': lang});
+
+const LanguageSelectBar = ({path, lang}) => (
+  <div className="coa-LanguageSelectBar">
+    <div className="wrapper">
```

Comment:
just an FYI, container-fluid is a class that auto adds 2rems of padding to the left and right of a container div.
I've been using it with wrapper to always ensure there's the same size padding on the left and right of container divs and same size max-widths.
So like
`<div class="container-fluid wrapper"> ...</div>`

Reply:
👍 👍 👍